daemon/kill.go

```diff
@@ -55,7 +27,7 @@ func (daemon *Daemon) ContainerKill(job *engine.Job) error {
 		if err := container.KillSig(int(sig)); err != nil {
 			return fmt.Errorf("Cannot kill container %s: %s", name, err)
 		}
-		// FIXME: Add event for signals
+		container.LogEvent("kill signal:" + strconv.Itoa(int(sig)))
```

Comment:
I think it should be sort of separate PR.
I don't know how to post signal to events - it is sorta break events format.